pypowsybl_jupyter/explorer.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2022, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from IPython.display import display
+from pypowsybl.network import Network
+from pypowsybl_jupyter.svg import display_svg
+
+import ipywidgets as widgets
+
+
+def network_explorer(network: Network):
+    """
+    Creates a basic explorer widget for this network.
+    """
+
+    vl_input = widgets.Text(
+        value='',
+        placeholder='Voltage level ID',
+        description='Voltage level:',
+        disabled=False,
+        continuous_update=True
+    )
+
+    found = widgets.SelectMultiple(
+        options=[],
+        value=[],
+        rows=10,
+        description='Found',
+        disabled=False
+    )
+
+    def on_text_changed(d):
+        vls = network.get_voltage_levels(attributes=[])
+        vls = vls[vls.index.str.startswith(d['new'])]
+        found.options = vls.index
+
+    vl_input.observe(on_text_changed, names='value')
+
+    button = widgets.Button(description="Display voltage level")
```

Comment:
I think we should remove this button and instead observe with a callback the selection in the list
```python
found.observe(on_select_changed, names='value')
```
This would be simpler to use.